packages/ebayui-core/.storybook/main.ts

```diff
@@ -19,7 +19,7 @@ const rawMarkdown = {
 export default {
     stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|mjs|ts|tsx)"],
     framework: "@storybook/marko-vite",
-    addons: ["@storybook/addon-essentials", "@storybook/addon-a11y"],
+    addons: ["@storybook/addon-a11y", "@storybook/addon-docs"],
```

Comment:
Is removing addon-essentials intentional?

Reply:
@HenriqueLimas it is enabled be default in sb9